tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -338,30 +339,85 @@ export interface moveOrCopyResourceArgs {
   resource: string
   newLocation: string
   action: 'copy' | 'move'
+  method: string
+}
+export const pasteResource = async (
+  args: Omit<moveOrCopyResourceArgs, 'action' | 'method'>
+): Promise<void> => {
+  const { page, resource, newLocation } = args
+
+  await page.locator(breadcrumbRoot).click()
+  const newLocationPath = newLocation.split('/')
+
+  for (const path of newLocationPath) {
+    if (path !== 'Personal') {
+      await clickResource({ page, path: path })
+    }
+  }
+
+  await page.locator(clipboardBtns).first().click()
+  await waitForResources({
+    page,
+    names: [resource]
+  })
 }
 
 export const moveOrCopyResource = async (args: moveOrCopyResourceArgs): Promise<void> => {
-  const { page, resource, newLocation, action } = args
+  const { page, resource, newLocation, action, method } = args
   const { dir: resourceDir, base: resourceBase } = path.parse(resource)
 
   if (resourceDir) {
     await clickResource({ page, path: resourceDir })
   }
 
-  await page.locator(util.format(resourceNameSelector, resourceBase)).click({ button: 'right' })
-  await page.locator(util.format(filesAction, action)).first().click()
-  await page.locator(breadcrumbRoot).click()
-
-  if (newLocation !== 'Personal') {
-    await clickResource({ page, path: newLocation })
-  }
+  switch (method) {
+    case 'menu': {
+      await page.locator(util.format(resourceNameSelector, resourceBase)).click({ button: 'right' })
+      await page.locator(util.format(filesAction, action)).first().click()
+      await pasteResource({ page, resource: resourceBase, newLocation })
+      break
+    }
```

Comment:
```suggestion
    default: {
      await page.locator(util.format(resourceNameSelector, resourceBase)).click({ button: 'right' })
      await page.locator(util.format(filesAction, action)).first().click()
      await pasteResource({ page, resource: resourceBase, newLocation })
      break
    }
```

Reply:
ohh no.
what if somebody tries to use `sidebar` but instead misspells it. e.g. `...copies the following resource using sidebal`.
Now, I think the default should throw an error.

Reply:
@saw-jan 
```
   ? And "Alice" copies the following resource using sidebal panel
       | resource    | to          |
       | sidebar.txt | PARENTCopy2 |
       Undefined. Implement with the following snippet:

         Given('{string} copies the following resource using sidebal panel', async function (this: World, string: any, dataTable: any): Promise<void> {
           const { feature, actorsEnvironment, usersEnvironment, filesEnvironment } = this

           await new Promise(resolve => setTimeout(resolve, 10))
         });

```